crates/orderbook/src/order_quoting.rs

```diff
@@ -111,6 +113,7 @@ pub struct QuoteParameters {
     pub side: OrderQuoteSide,
     pub from: H160,
     pub app_data: AppId,
+    pub signing_scheme: SigningScheme,
```

Comment:
We probably also need to store this with the quote (so that we don't create a `presign` quote and use it for a standard order).